BAZProjectC1/List/Model/Movie.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Movie.swift
+//  BAZProjectC1
+//
+//
+
+import Foundation
+
+struct Movie {
+    let id: Int
+    let title: String
+    let descriptionMovie: String
+    let poster_path: String
```

Comment:
La convención para nombres de propiedades en swift es `lowerCammelCase`, considera cambiar este nombre:
```suggestion
    let posterPath: String
```